injector_test.go

```diff
@@ -408,3 +427,20 @@ func BenchmarkProcess(b *testing.B) {
 		}
 	})
 }
+
+func assertJSONEqual(t *testing.T, expected, actual string) {
+	var expectedJSON, actualJSON interface{}
+
+	if err := json.Unmarshal([]byte(expected), &expectedJSON); err != nil {
+		t.Errorf("Expected ('%s') is not a valid json: '%s'", expected, err.Error())
```

Comment:
это не ошибка теста, это ошибка его составления, поэтому тут паника, но не крит если оставишь t.Fatal

и я бы точно убрал вывод actual / expected из этой ошибки - они зачастую очень большие и лишь затруднят разбор

```suggestion
		panic("expected value is not a valid json: " + err.Error())
```

Reply:
Супер